phoenix-core/src/main/java/org/apache/phoenix/monitoring/GlobalClientMetrics.java

```diff
@@ -80,6 +81,7 @@
     GLOBAL_MUTATION_BYTES(MUTATION_BYTES),
     GLOBAL_MUTATION_COMMIT_TIME(MUTATION_COMMIT_TIME),
     GLOBAL_MUTATION_BATCH_FAILED_COUNT(MUTATION_BATCH_FAILED_SIZE),
+    GLOBAL_MUTATION_PHASE_THREE_FAILURE_COUNT(PHASE_THREE_FAILURE_SIZE),
```

Comment:
Could we have a more descriptive name than PHASE_THREE_FAILURE_COUNT, which is more intuitive if you don't already know a great deal about the indexing pipeline? GLOBAL_MUTATION_INDEX_COMMIT_FAILURE_COUNT maybe? @kadirozde , what do you think would be a good name?

Reply:
@gjacoby126,  GLOBAL_MUTATION_INDEX_COMMIT_FAILURE_COUNT sounds good to me.